python/datasources/cdc_hiv.py

```diff
@@ -94,144 +48,110 @@ def get_table_name():
 
     def upload_to_gcs(self, gcs_bucket, **attrs):
         raise NotImplementedError(
-            'upload_to_gcs should not be called for CDCHIVData'
-        )
+            'upload_to_gcs should not be called for CDCHIVData')
 
     def write_to_bq(self, dataset, gcs_bucket, **attrs):
-        for geo_level in [constants.COUNTY_LEVEL, constants.STATE_LEVEL, constants.NATIONAL_LEVEL]:
+
+        for geo_level in [COUNTY_LEVEL, NATIONAL_LEVEL, STATE_LEVEL]:
+            alls_df = generate_df_from_data_dir(f'hiv-{geo_level}-all.csv')
+
             for breakdown in [std_col.AGE_COL, std_col.RACE_OR_HISPANIC_COL, std_col.SEX_COL]:
                 table_name = f'{breakdown}_{geo_level}'
+                df = self.generate_breakdown_df(breakdown, geo_level, alls_df)
 
-                df = self.generate_breakdown_df(breakdown, geo_level)
-
-                float_cols = [std_col.HIV_POPULATION_PCT,
-                              std_col.HIV_DIAGNOSES,
+                float_cols = [std_col.HIV_DIAGNOSES,
+                              std_col.HIV_DIAGNOSES_PCT_INEQUITY,
+                              std_col.HIV_DIAGNOSES_PCT_SHARE,
                               std_col.HIV_DIAGNOSES_PER_100K,
-                              std_col.HIV_DIAGNOSES_PCT_SHARE
-                              ]
-
-                column_types = gcs_to_bq_util.get_bq_column_types(
-                    df, float_cols=float_cols)
-
-                gcs_to_bq_util.add_df_to_bq(
-                    df, dataset, table_name, column_types=column_types)
-
-    def generate_breakdown_df(self, breakdown, geo_level):
-        COL_NAME = std_col.COUNTY_NAME_COL if geo_level == constants.COUNTY_LEVEL else std_col.STATE_NAME_COL
-        FIPS = std_col.COUNTY_FIPS_COL if geo_level == constants.COUNTY_LEVEL else std_col.STATE_FIPS_COL
-
-        cols_std = {
-            'Geography': COL_NAME,
-            'FIPS': FIPS,
+                              std_col.HIV_POPULATION_PCT,
+                              std_col.HIV_DIAGNOSES_PCT_INEQUITY]
+                column_types = gcs_to_bq_util.get_bq_column_types(df,
+                                                                  float_cols=float_cols)
+                gcs_to_bq_util.add_df_to_bq(df,
+                                            dataset,
+                                            table_name,
+                                            column_types=column_types)
+
+    def generate_breakdown_df(self, breakdown: str, geo_level: str, alls_df: pd.DataFrame):
```

Comment:
maybe add a doc string here? I think in general we are trying to add both type hints and doc strings to all our functions just for future ease of use and documentation